meson.build

```diff
@@ -207,6 +207,10 @@ if get_option('gstreamer')
              dependency('gstreamer-allocators-1.0'),]
 endif
 
+if build_machine.system() == 'freebsd'
+  epoll_shim_dep = dependency('epoll-shim', required: true)
+endif
+
```

Comment:
You leave this undefined unless the platform is freebsd. I.e. if you try to compile it on any other, it gonna break the build. You probably want something like this instead:

```meson
epoll_shim_dep = (build_machine.system() == 'freebsd') \
  ? dependency('epoll-shim', required: true) : dependency('', required: false)
```

Reply:
I also think you may want to move the description about epoll-shiim that you have in commit message here in comment instead.